README.md

```diff
@@ -1,87 +1,84 @@
 
-This is built to run as an always-on daemon, fetching data every minute from
-Cloudflare and posting it to Datadog. Cloudflare won't necessarily return new
-data every minute and the script will make sure to filter the ones already sent
-since its start.
+This is built to run as an always-on daemon, fetching data on configurable
+intervals from Cloudflare and posting it to Datadog. Cloudflare won't
+necessarily return new data every time and the script will make sure to filter
+the ones already sent since its start.
 
 Data are posted to Datadog through its HTTP API which allows combinations of
 timestamps and values, in contrary to statsd which does not -by design- allow
 sending timestamps.
 
 
-## Instructions
+## Setup
 Populate the following variables in the environment or a `.env` file.
 
-  `DATADOG_API_KEY`: Datadog API Key
-
-  `DATADOG_APP_KEY`: Datadog App Key
+ - `ZONE`: Cloudflare Zone ID or alternatively use `DOMAIN`
+ - `DOMAIN`: Domain or alternatively use `ZONE`
+ - `CF_API_EMAIL`: Cloudflare auth email
+ - `CF_API_KEY`: Cloudflare auth key
+ - `FETCH_INTERVAL`: Fetch every X minutes. (Default: 1)
+ - `SINCE`: Since when to fetch data from. (Default: -360)
 
-  `STATS_KEY_PREFIX`: Prefix for the metrics sent in Datadog.
+   Note:
+   * Free plans can only get up to 60 min. (Since: -1440)
+   * Pro & Business plans can only up to 15 min. (Since: -360)
+   * Enterprise plans can get up to 1 min. (Since: -30)
 
-  `AUTH_EMAIL`: Cloudflare auth email
+ - `DATADOG_API_KEY`: Datadog API Key
+ - `DATADOG_APP_KEY`: Datadog App Key
+ - `STATS_KEY_PREFIX`: Prefix for the metrics sent in Datadog.
+ - `TAGS`: List of comma separated Datadog Tags
 
-  `AUTH_KEY`: Cloudflare auth key
+ - `LOG_LEVEL`: Script log level. (Default: INFO)
+ - `DEAD_MANS_SNITCH_URL`: A url to ping after every successful submission to
+   Datadog. You can use a service
+   like [deadmansnitch](http://deadmanssnitch.com/)
+   or [healthchecks](http://healthchecks.io/)
 
-  `SINCE`: Cloudflare Analytics (based on Plan)
+ - `TIMEZONE`: Docker only. Set the timezone of the container. (Default: UTC)
 
-  * Free plan can only get 1 hour period analytics (Since: -1440)
-  * Pro & Business plans can only get 15 min. periods (Since: -360)
-  * Enterprise can get up to 1 min. period analytics (Since: -30)
- 
-  `TAGS`: List of comma separated Datadog Tags
+### Getting Zone ID
 
-  `ZONE`: Cloudflare Zone
+You can provide `DOMAIN` and let the script figure out the Zone ID for you.
+Otherwise you can directly provide `ZONE` in the environment.
 
-How to get Cloudflare zone id:
+To get the Zone IDs for your domains populate `CF_API_EMAIL` and `CF_API_KEY` and
+run:
 
-Populate `AUTH_EMAIL` and `AUTH_KEY` parameters, then:
 ```
 make build
 make shell
-apk add --no-cache jq curl
-curl https://api.cloudflare.com/client/v4/zones?name=example.com \
- -H "X-Auth-Email: $AUTH_EMAIL" -H "X-Auth-Key: $AUTH_KEY" \
- | jq -r .result[].id
+./getZoneIDs.py
```

Comment:
yeah, I noticed that yesterday :p - didn't see that when I set up the integration a while ago, but in the new PR @glogiotatidis added a small script for this anyway.